src/modules/dashboard/DashboardTopPanel.tsx

```diff
@@ -237,6 +245,23 @@ export const DashboardTopPanel = () => {
               >
                 <Trans>Claim</Trans>
               </Button>
+              {connected && (
+                <AddTokenDropdown
+                  poolReserve={{
+                    ...reserves[1],
+                    symbol: 'OG Points',
```

Comment:
We don't have OG Points in `reserves`, using the splatted reserves to appease 'pool reserve' prop but only uses overridden values

Reply:
casted